### PR TITLE
[15.0][MIG] account_sale_stock_report_non_billed: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT40223

@chienandalu @CarlosRoca13 please review!

### CODE REVIEW

File: account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py [hunk 1]
```diff
@@ -21,7 +22,7 @@ def get_picking_done_so(self):
             and x.state in ("confirmed", "assigned", "partially_available")
         )
         picking.move_line_ids.write({"qty_done": 2})
-        picking.action_done()
+        picking._action_done()
```
Comment (chienandalu) [inline account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py]: change it to `button_validate` in all the cases

Comment (stefan-tecnativa) [reply account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py]: Change done

Comment (stefan-tecnativa) [reply account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py]: Changing all the cases for `button_validate` causes a test to fail. If in that case if I put `_action_done` the test no longer fails. Can it be a problem of my local installation, that there is something that causes conflicts and that is why it fails in something so specific?

Comment (chienandalu) [reply account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py]: Well, that's interesting as that's the action that the users will be triggering.  You have to find out what's the difference that makes the tests fail. Do the same test case work if you try it out by UI?

Comment (stefan-tecnativa) [reply account_sale_stock_report_non_billed/tests/test_account_sale_stock_report_non_billed.py]: Now it should be correct.

**Review by chienandalu — APPROVED** [review]
Great!

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**